### PR TITLE
bump gLang version to 1.19

### PR DESCRIPTION
**Pull request by roytman**
All supported Kubernets distributions (1.23 1.24) built with goLang 1.19, therefore we have to bump our project to 1.19 too.

/fixes https://github.com/fybrik/fybrik/issues/1899

Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

File: pkg/taxonomy/compile/transform.go [hunk 1]
```diff
@@ -17,7 +17,8 @@ const nameKey = "name"
 // transform applies transformations over an input document to make it structural.
 // It requires a base document and the document to transform (mutable).
 // The codegenTarget can be set to true to apply more transformations to make the document
-//  more suitable for code generation tools.
+//
```
Comment (shlomitk1) [inline pkg/taxonomy/compile/transform.go:20]: Why are there empty lines in a middle of a comment?

**Comment by shlomitk1** [comment]
Please add a description. Why is this PR required? What issue does it solve?

**Review by revit13 — APPROVED** [review]

File: pkg/taxonomy/compile/transform.go (hunk 1 above)
Comment (roytman) [reply pkg/taxonomy/compile/transform.go]: `go fmt` changed his comment formatting, as result all the changes in comments. 
Fixed this new line separation.

Comment (shlomitk1) [reply pkg/taxonomy/compile/transform.go]: There should not be a separation in this or other cases, just indent the last line correctly - it has an extra space. In other cases, there is an extra tab. All comments need to start at the same indentation.

Comment (roytman) [reply pkg/taxonomy/compile/transform.go]: No, fmt adds a new line before lint tags. 
Please check now.

File: pkg/optimizer/datapath_csp.go [hunk 2]
```diff
@@ -152,8 +152,9 @@ func (dpc *DataPathCSP) addInterface(intfc *taxonomy.Interface) {
 
 // This is the main method for building a FlatZinc CSP out of the data-path parameters and constraints.
 // Returns a file name where the model was dumped
-// NOTE: Minimal index of FlatZinc arrays is always 1. Hence, we use 1-based modeling all over the place to avoid confusion
-//       The only exception is with interfaces (0 means nil)
+//
```
Comment (shlomitk1) [inline pkg/optimizer/datapath_csp.go:155]: Remove the empty line

File: pkg/vault/vault_impl.go [hunk 3]
```diff
@@ -98,7 +98,8 @@ func (c *Connection) RemovePolicyFromIdentity(identity, policyName, auth string)
 // WritePolicy stores in vault the policy indicated.  This can be associated with a vault token or
 // an authentication identity to ensure proper use of secrets.
 // Example policy: "path \"identities/test-identity\" {\n	capabilities = [\"read\"]\n }"
-// 		NOTE the line returns and the tab.  Without them it fails!
+//
```
Comment (shlomitk1) [inline pkg/vault/vault_impl.go:101]: Remove the empty line

File: pkg/optimizer/datapath_csp.go (hunk 2 above)
Comment (roytman) [reply pkg/optimizer/datapath_csp.go:155]: I think it is better to separate the `Note:`

**Comment by roytman** [comment]
> Please add a description. Why is this PR required? What issue does it solve?

done

File: pkg/vault/vault_impl.go (hunk 3 above)
Comment (roytman) [reply pkg/vault/vault_impl.go:101]: again, I tried to separate Note with the empty line

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**